pkgs/development/python-modules/hydrogram/default.nix

```diff
@@ -25,6 +25,11 @@ buildPythonPackage rec {
     hash = "sha256-QpweUDCypTxOOWL08gCUuMgbuE4130iNyxRpUNuSBac=";
   };
 
+  patches = [
+    # adapted from https://github.com/hydrogram/hydrogram/commit/7431319a1d990aa838012bd566a9746da7df2a6e
+    ./fix-async-in-test.patch
```

Comment:
I think you can do `fetchpatch2 { url = "https://github.com/hydrogram/hydrogram/commit/7431319a1d990aa838012bd566a9746da7df2a6e.patch"; hash = "sha256-U0x9a/EbSwjMUmSjQGZ+4zhQwy71bmVUQfUwQceIoy0="; }`